core/jazz_usermanagement/test/test.js

```diff
@@ -1,11 +1,494 @@
 const assert = require('chai').assert;
+const expect = require('chai').expect;
+const should = require('chai').should();
+const awsContext = require('aws-lambda-mock-context');
+const  AWS  =  require('aws-sdk-mock');
+const sinon = require('sinon');
 const index = require('../index');
+const logger = require("../components/logger.js");
+const errorHandlerModule = require("../components/error-handler.js");
+const scmFactory = require("../scm/scmFactory.js");
+const configModule = require("../components/config.js");
+const responseObj = require("../components/response.js");
+const rp = require('request-promise-native');
+
+describe('User Management', function () {
+	//Setting up default values for the aws event and context needed for handler params
+	beforeEach(function () {
+		event = {
+			"method": "POST",
+			"stage": "test",
+			"resourcePath": "test/service/user/reset",
+			"body": {
+				"username": "username",
+				"userid": "username",
+				"verificationCode": "123",
+				"email": "abc@xyz.com"
+			}
+		};
+		context = awsContext();
+		callback = (value) => {
+			return value;
+		};
+		config = configModule.getConfig(event, context);
+	});
+
+	it('should throw error validateCreaterUserParams', function () {
+		config.required_fields = {
+			'userid': 'userid',
+			'email': 'email',
+			'verificationCode': 'verificationCode',
+			'param': 'param'
+		}
+		let result = index.validateCreaterUserParams(config, event.body);
```

Comment:
no need for the variable. Just directly call the function and catch on that

Reply:
resolved.